server/src/weaverbird/pipeline/pipeline.py

```diff
@@ -184,6 +194,79 @@ def dict(self, *, exclude_none: bool = True, **kwargs) -> dict:
 ]
 
 
+def _remove_void_from_combo_condition(
+    condition: ConditionComboAnd | ConditionComboOr | None, combo: str
```

Comment:
```suggestion
    condition: ConditionComboAnd | ConditionComboOr, combo: str
```

Why should this receive `None` as a parameter ?

Reply:
my code editor... in reality, there is no need for that None, removed !